test/jit/test_complex.py

```diff
@@ -130,3 +130,33 @@ def fn(x: complex):
 
         for val in complex_vals:
             self.checkScript(fn, (val, ))
+
+    def test_cmath_constants(self):
+        def checkCmathConst(const_name, ret_type="float"):
+            funcs_template = dedent('''
+                def func():
+                    # type: () -> {ret_type}
+                    return cmath.{const}
+                ''')
+
+            funcs_str = funcs_template.format(const=const_name, ret_type=ret_type)
+            scope = {}
+            execWrapper(funcs_str, globals(), scope)
+            cu = torch.jit.CompilationUnit(funcs_str)
+            f_script = cu.func
+            f = scope['func']
+
+            res_python = f()
+            res_script = f_script()
+
+            if res_python != res_script:
+                msg = ("Failed on {const_name}. Python: {res_python}, Script: {res_script}"
+                    .format(const_name=const_name, res_python=res_python, res_script=res_script))
+                self.assertEqual(res_python, res_script, msg=msg)
+
+        float_consts = ['pi', 'e', 'tau', 'inf', 'nan']
+        complex_consts = ['infj', 'nanj']
```

Comment:
Is there any way to define these using `cmath` so that new constants that are added get tested automatically?

Unlikely scenario but I thought I'd ask.